client/src/components/v2/CameraRecording.js

```diff
@@ -25,31 +26,43 @@ import styles from './CameraRecording.module.css';
  * @returns {React.Component<CameraRecordingProps>} Component
  */
 export default function CameraRecording({ devices }) {
+  const status = useVideoFeedStatus();
+
+  // Check if at least one camera's video feed is online
+  const canRecord = Object.keys(status).reduce((acc, device) => !!status[device]?.online || acc, false);
+
   return (
     <Card>
       <Card.Body>
         <Card.Title>Recording Controls</Card.Title>
         <Row className={styles.row}>
-          {devices.map((device) => (
-            <Col className={styles.col} key={device} sm={6}>
-              <Card.Subtitle>{getPrettyDeviceName(device)}</Card.Subtitle>
-              <CameraRecordingStatus device={device} />
-            </Col>
-          ))}
+          {devices.map((device) => {
+            return (
+              <Col className={styles.col} key={device} sm={6}>
+                <Card.Subtitle>
+                  <OnlineIndicator className={styles.indicator} online={!!status[device]?.online} />
```

Comment:
This CSS isn't being applied for some reason. Send help...

Reply:
I think className only works on "native" elements, since it's just a normal prop. You can wrap this in a div and apply the class to that instead

Reply:
Done
d4010a5